src/test/java/com/google/javascript/clutz/partial/platform_goog_require.d.ts

```diff
@@ -1,7 +1,7 @@
 declare namespace ಠ_ಠ.clutz.module$exports$platform$goog$require {
   class C extends C_Instance {
   }
-  class C_Instance extends ಠ_ಠ.clutz.goog.events.EventTarget {
+  class C_Instance extends ಠ_ಠ.clutz.module$exports$goog$events$EventTarget {
```

Comment:
We have to fix the test by adding a simple depgraph for it, or just mocking it. Otherwise, this is a bad test with this change.

Reply:
Added a comment that that's orthogonal to the issue of ambiguity with GlobalEventTarget.